source/features/easy-toggle-commit-messages.tsx

```diff
@@ -18,18 +18,31 @@ function toggleCommitMessage(event: DelegateEvent<MouseEvent>): void {
 		return;
 	}
 
-	$('.ellipsis-expander', event.delegateTarget)?.dispatchEvent(
+	$('[data-testid="commit-row-show-description-button"]', event.delegateTarget)?.dispatchEvent(
```

Comment:
I'm not sure why the function was split in two. Both `delegate` and `$()` can accept an array of selectors. So this line should be:

```suggestion
	$([
		'[data-testid="commit-row-show-description-button"]', // `isCommitList` or whatever
		'[data-testid="latest-commit-details-toggle"]', // list all applicable detections for each 
	], event.delegateTarget)?.dispatchEvent(
```

and then the `delegate` array should be restored

Reply:
I mean here we will have two situations. The toggle buttons' class has changed.

```ts
$('[data-testid="latest-commit-details-toggle"]', event.delegateTarget)?.dispatchEvent(
	new MouseEvent('click', {bubbles: true, altKey: event.altKey}),
);

$('[data-testid="commit-row-show-description-button"]', event.delegateTarget)?.dispatchEvent(
	new MouseEvent('click', {bubbles: true, altKey: event.altKey}),
);
```